code/game/turfs/open/glass.dm

```diff
@@ -13,8 +13,11 @@
 	barefootstep = FOOTSTEP_HARD_BAREFOOT
 	clawfootstep = FOOTSTEP_HARD_CLAW
 	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	var/stateDamage = "glass"
 
 /turf/open/floor/glass/Initialize()
+	if (!broken_states)
+		broken_states = string_list(list("[stateDamage]-damaged1", "[stateDamage]-damaged2", "[stateDamage]-damaged3", "[stateDamage]-damaged4", "[stateDamage]-damaged5"))
```

Comment:
Does this have to be hardcoded on `Initialize()`?
I'd prefer this was manually setup on the definition, chiefly because this is not very expansible.

Reply:
i believe so, the initialize in the basetype/parent creates that list directly in its initialization, so i need to override its initialization